src/status_im2/contexts/communities/menus/community_options/view.cljs

```diff
@@ -82,19 +82,19 @@
   {:icon                :i/log-out
    :label               (i18n/label :t/leave-community)
    :accessibility-label :leave-community
-
    :danger?             true
    :on-press            #(rf/dispatch [:bottom-sheet/show-sheet
                                        {:content        (constantly [leave-menu/leave-sheet id])
                                         :content-height 400}])})
 
 (defn cancel-request-to-join
   [id]
-  {:icon                :i/block
-   :label               (i18n/label :t/cancel-request-to-join)
-   :accessibility-label :cancel-request-to-join
-   :danger?             true
-   :on-press            #(js/alert (str "implement action" id))})
+  (let [request-id (rf/sub [:communities/my-pending-request-to-join id])]
```

Comment:
I'd rename `id` to `community-id`, since it's not immediately clear from the function (I believe that's `id` ->  `community-id` passed as parameter, correct?)

Reply:
Yep, good point